cmd/argo/commands/cluster/rm.go

```diff
@@ -0,0 +1,44 @@
+package cluster
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/argoproj/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/argoproj/argo/cmd/argo/commands/client"
+)
+
+func NewRMCommand() *cobra.Command {
+	return &cobra.Command{
+		Use: "rm CLUSTER_NAME",
```

Comment:
`delete`? Consistent with `argo {template,cron} delete`

Reply:
Perhaps not since it doesn't delete the cluster, but removes it from being tracked

Reply:
good point - I copied from Argo CD - but @jessesuen explained a reason for this to me - I think it was that you were not actually deleting the cluster, just removing it from from the config - but I feel not very strongly at all about this